tools/devnet.dockerfile

```
@@ -7,6 +7,11 @@ ARG COMMIT_HASH_FIL=""
 ARG REPO_FIL=https://github.com/filecoin-project/lotus
 ARG NODEPATH=/lotus
 
+# set BRANCH_PROXY or COMMIT_HASH_PROXY
+ARG BRANCH_PROXY=master
+ARG COMMIT_HASH_PROXY=""
+ARG REPO_PROXY=https://github.com/Zondax/rosetta-filecoin-proxy.git
```

```suggestion
ARG REPO_PROXY=https://github.com/Zondax/filecoin-indexing-rosetta-proxy.git
```
We're using this other repo for the rosetta proxy, which is basically the same as `rosetta-filecoin-proxy` but having this in another repo allows us the make changes/tweaks without interfering with other projects using the original proxy project as dependency

---

I thought that repo was intended to be use for indexing, and It has some specific changes for other purposes.